### ISSUE TITLE
Request threshold too small and hit "429: Too Many Requests."

### ISSUE THREAD

**Issue by t1gu1** [pastelsky/bundlephobia#886]
**Describe the bug**
I just compared few packages and I hit the `429: Too Many Requests.` error..

**To Reproduce**
Not that sure. I open few packages (Something like 5) and I navigate through versions of the package.

**Expected behavior**
Don't hit the `429: Too Many Requests.` that rapidly.

**Comment by t1gu1** [comment]
Oh and I click on a version when it was telling "click to build", maybe that's the cause?

![image](https://github.com/user-attachments/assets/4ed5a8b3-7417-4a45-a485-0a7fe25ccaa0)

**Comment by jasperdunn** [comment]
I didn't click that button but got the same thing. Also only looked at about three or four packages.

**Comment by pmooney-socraticworks** [comment]
I got the same thing just comparing a few packages on their website.

**Comment by Vanilagy** [comment]
Yup, same, just compared like 6 packages and hit this. Not sure what the cooldown is, either
